src/main/java/com/adobe/jenkins/github_pr_comment_build/GitHubPullRequestCommentCause.java

```diff
@@ -7,13 +7,25 @@
  */
 public final class GitHubPullRequestCommentCause extends Cause {
     private final String commentUrl;
+    private final String commentBody;
 
     /**
      * Constructor.
      * @param commentUrl the URL for the GitHub comment
      */
     public GitHubPullRequestCommentCause(String commentUrl) {
         this.commentUrl = commentUrl;
+        this.commentBody = "Unknown";
```

Comment:
I feel that this is not correct. it really should be an empty string. Otherwise how could someone tell the difference between "Unknown" as a comment and this default case?

Reply:
Alternatively, you could take out this constructor since I believe it's not used anymore, although people outside of the plugin may have depended on it. However, they can also just update their usage if they are using internal classes, so I think I'm leaning towards just removing this version of the constructor.

Reply:
Thank you for getting the build working again. I've gone ahead and removed the old constructor. I wasn't sure at first how backwards compatible things needed to be, but agree that it's completely unnecessary now internally.